### PR TITLE
Final reviewer changes

### PR DESCRIPTION
**Pull request by jeromekelleher**
I've added my attempt at a paragraph explaining why we should treat the results of neutral sims with caution. Any thoughts @andrewkern @sgravel @petrelharp

I backed out your earlier changes @andrewkern as it didn't really fit into the narrative of the final paragraph, and disrupted another point we were trying to make there.

### CODE REVIEW

**Comment by jeromekelleher** [comment]
@KLohse, you might have some thoughts here on how to meet reviewer's not unreasonable point. Do you think a short para at the end of the introduction stating that nuetral sims are limited is OK? How do these references look to you?

File: response-to-reviewers.tex [hunk 1]
```diff
@@ -620,9 +620,9 @@ \section*{Response to the reviewers}
 \end{point}
 
 \begin{reply}
-We agree that neutral simulations are an approximation, and care needs to
```
Comment (KLohse) [inline response-to-reviewers.tex:623]: We agree that neutral simulations are approximation, and care needs to be exercised in their interpretation and use. We have added text to the introduction to emphasise this point. However, as we have argued above, major methodological advances of the type that you discuss (which we agree are extremely important!) are beyond the scope of the current manuscript. While the question of how to best approximate background selection in a coalescent framework is a topic of ongoing research (e.g. Nocloaisen & Desai 2013, Zeng & Charlesworth 2011), we would argue that being able to simulate ancestry under more realistic non-neutral null models (e.g. condition on maps of background selection) will only be useful if such simulations can be both efficient and embedded in realistic models of other biological processes, many of which we have implemeted in this version of msprime.

File: paper.tex [hunk 2]
```diff
@@ -279,9 +279,15 @@ \section*{Introduction}
...
-% standard piece of computational infrastructure for the genetics community.
+The efficiency of coalescent simulations depends
+crucially on the assumption of neutrality, and it is important
+to note that there are many situations in which this will be a poor approximation
+of biological reality~\citep{johri2021statistical}.
+In particular the existence of pervasive background
+selection in many
+species~\citep{charlesworth1993effect,charlesworth1995pattern,charlesworth2021effects}
+means that care must be taken to ensure that the results of
+neutral simulations are appropriate for the study in question.
```
Comment (KLohse) [inline paper.tex:291]: -> In particular, background selection has been shown to affect genomewide sequence variation in a wide range of species  ~\citep{charlesworth1993effect,charlesworth1995pattern,charlesworth2021effects}. Thus care must be taken to ensure that the results of neutral simulations are appropriate for the question and genomic partition under study. While approximating background selection is not features will only be useful if such simulations can be both efficient and embedded in realistic models of other biological processes, many of which we have implemeted in this version of msprime.

**Comment by KLohse** [comment]
@jeromekelleher Reviewer 3 is a real eeyore! I bet it's one of the BGS silverbacks on the Johri et al MS :-) . I think your addition of a cautionary sentence at the end of the intro is all we need. I've made some minor tweaks to this and the response.

**Review by KLohse — COMMENTED** [review]
@jeromekelleher. Could you add:  "GB and KL are supported by an ERC starting grant (ModelGenomLand 757648) to KL."

**Comment by jeromekelleher** [comment]
Thanks Konrad, all done!

> I bet it's one of the BGS silverbacks on the Johri et al MS :-)

Haha, I can't help imaging all of them sitting around in the jungle somewhere grunting at each other now! If I could use photoshop I'd be pasting faces onto pictures of gorillas right now...

**Comment by jeromekelleher** [comment]
Merging this so we're up to date - we may still do some tweaks though

**Pull request merged**